lib/state/appState.js

```diff
@@ -89,6 +89,17 @@ appState.getInitial = function() {
   };
 };
 
+appState.hideUnavailableDevices = function() {
+  var currentOS = this.app.device._os; // "mac", "win", "android", "cros", "linux", or "openbsd"
```

Comment:
I had to trace this through to figure out that `_os` gets added as a property of `device` in `lib/core/device.js`...it seemed an odd path, and I'd certainly be in favor of promoting the `_os` attribute up a level to `this.app._os` if you agree.